workflows/tfactivity.nf

```diff
@@ -139,7 +141,7 @@ workflow TFACTIVITY {
     FIMO(
         fasta,
         RANKING.out.tf_total_ranking,
-        PEAKS.out.enhancers,
+        PEAKS.out.tf_bindings,
```

Comment:
Not something that was introduced with this PR, but still relevant:
This will only work if BAM files are available, as otherwise ROSE won't be used. Should we not include all the regions that were used as for the affinity calculation here?